examples/ansible/kubeone/roles/kubeone_config/defaults/main.yml

```diff
@@ -0,0 +1,92 @@
+# Copyright 2019 The KubeOne Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kubeone_versions_kubernetes: 1.14.1
+kubeone_clusterNetwork_podSubnet: 10.244.0.0/16
+kubeone_clusterNetwork_serviceSubnet: 10.96.0.0/12
+kubeone_clusterNetwork_serviceDomainName: cluster.local
+kubeone_clusterNetwork_nodePortRange: 30000-32767
+kubeone_clusterNetwork_cni_provider: canal
+kubeone_clusterNetwork_cni_encrypted: false
+kubeone_cloudProvider_name: aws
+kubeone_cloudProvider_external: false
+kubeone_cloudProvider_cloudconfig: ""
+kubeone_apiEndpoint_host: ""
+kubeone_apiEndpoint_port: 0
+kubeone_features_podSecurityPolicy_enable: false
+kubeone_features_dynamicAuditLog_enable: false
+kubeone_features_metricsServer_enable: true
+kubeone_machineController_deploy: true
+kubeone_machineController_provider: ""
+kubeone_proxy_http: ""
+kubeone_proxy_https: ""
+kubeone_proxy_noProxy: ""
+kubeone_openidConnect_enable: false
+kubeone_openidConnect_config_issuerUrl: ""
+kubeone_openidConnect_config_clientId: "kubernetes"
+kubeone_openidConnect_config_usernameClaim: "sub"
+kubeone_openidConnect_config_usernamePrefix: "oidc:"
+kubeone_openidConnect_config_groupsClaim: "groups"
+kubeone_openidConnect_config_config_groupsPrefix: "oidc:"
+kubeone_openidConnect_config_signingAlgs: "RS256"
+kubeone_openidConnect_config_requiredClaim: ""
+kubeone_openidConnect_config_caFile: ""
+kubeone_hosts: []
+# kubeone_hosts:
+# - publicAddress: '1.2.3.4'
+#   privateAddress: '172.18.0.1'
+#   bastion: '4.3.2.1'
+#   bastionPort: 22
+#   sshPort: 22
+#   sshUsername: ubuntu
+#   sshPrivateKeyFile: '/home/me/.ssh/id_rsa'
+#   sshAgentSocket: 'env:SSH_AUTH_SOCK'
+kubeone_workers: []
+# kubeone_workers:
```

Comment:
This is not going to work out-of-box as VPC, SecurityGroups and subnets are missing. Should we add placeholder fields with a message that those fields should be populated manually?

Reply:
Have added packet example.

Reply:
Is this needed considering we have that in another file?